views/base.njk

```diff
@@ -5,6 +5,7 @@
 {% from 'checkboxes.njk' import checkBoxes as checkBoxes with context %}
 {% from 'buttons.njk' import formButtons as formButtons with context %}
 {% from 'input-file.njk' import fileInput as fileInput with context %}
+{% from 'medical-consent.njk' import medicalConsent as medicalConsent with context %}
```

Comment:
I wonder if since this is only used in the one place, maybe it should just be imported on that page? would reduce future collisions when merging from upstream.

Reply:
:+1: